ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOModelTestSuite.scala

```diff
@@ -623,4 +623,219 @@ class H2OMOJOModelTestSuite extends FunSuite with SharedH2OTestContext with Matc
       cvModels should be(null)
     }
   }
+
+  test("should return model summary") {
```

Comment:
nit: Can we add subjects to test names? E.g.:`H2OGBMModel`/`H2ODeepLearnigMOJOModel` should return model summary?